### ISSUE TITLE
What Does the Number 30 Mean in self.type_embedding = nn.Embedding(30, self._config.tf_d_model)?

### ISSUE THREAD

**Issue by sunbin1357** [YvanYin/GoalFlow#6]
What does the number 30 mean in `self.type_embedding = nn.Embedding(30, self._config.tf_d_model)`? I noticed that the code only uses the indices 0, 1, and 2. For example, in the function def encode_status_features(self, ego_agent_features):, the code uses:
```
    def encode_status_features(self, ego_agent_features):
        ego_features = self.history_encoder(ego_agent_features)

        ego_type_embedding = self.type_embedding(torch.as_tensor([[2]], device=ego_features.device))
        ego_type_embedding = ego_type_embedding.repeat(ego_features.shape[0],4,1)

        return ego_features, ego_type_embedding
```
Here, index 2 is used.

**Comment by frkmac3** [comment]
> [@sunbin1357](https://github.com/sunbin1357) Actually, 30 is the final trajectory dim after data augmentation. Every trajectory will be rotated 10 times, so the final dim is 3*10. You can refer to `normalize_xy_rotation` in `goalflow_model_traj.py`.

is it necessary to rotate 10 times during model infer?

**Comment by ZebinX** [comment]
@frkmac3 no, it's not necessary.
